waitress/rfc7230.py

```diff
@@ -33,12 +33,20 @@
 # field-content  = field-vchar [ 1*( SP / HTAB ) field-vchar ]
 # field-vchar    = VCHAR / obs-text
 
+# Errata from: https://www.rfc-editor.org/errata_search.php?rfc=7230&eid=4189
+# changes field-content to:
+#
+# field-content  = field-vchar [ 1*( SP / HTAB / field-vchar )
+#                  field-vchar ]
```

Comment:
I noticed the errata also changes `obs-fold` definition - which this PR doesn't seem to handle. Is that an oversight or fine?

Reply:
Waitress already handled it by allowing the optional whitespace at the end before the CRLF. Header field folding is also not handled by this regex at all, but is instead handled before the header is validated, so the regex doesn't see any folds at all.